scrapers/Spacebattles/Spacebattles/middlewares.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+# Define here the models for your spider middleware
+#
+# See documentation in:
+# http://doc.scrapy.org/en/latest/topics/spider-middleware.html
```

Comment:
See previous comment about this comment block (I won't say this on every file, I promise, just this last one).